tests/bluetooth/bsim_bt/edtt_ble_test_app/tests_scripts/ll.set1.llcp.test_list

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Oticon A/S
+# SPDX-License-Identifier: Apache-2.0
+
+LL/CON/ADV/BV-01-C
+LL/CON/ADV/BV-04-C
+LL/CON/ADV/BV-09-C
+LL/CON/ADV/BV-10-C
+LL/CON/INI/BV-01-C
+LL/CON/INI/BV-02-C
+LL/CON/INI/BV-06-C
+LL/CON/INI/BV-07-C
+LL/CON/INI/BV-08-C
+LL/CON/INI/BV-09-C
+LL/CON/INI/BV-10-C
+LL/CON/INI/BV-11-C
+LL/CON/INI/BV-12-C
+LL/CON/INI/BV-16-C
+LL/CON/INI/BV-17-C
+LL/CON/INI/BV-18-C
+LL/CON/INI/BV-19-C
+LL/CON/INI/BV-20-C
+LL/CON/INI/BV-21-C
+LL/CON/INI/BV-23-C
+LL/CON/INI/BV-24-C
+#LL/CON/MAS/BI-06-C #currently failing, to be investigated
+LL/CON/MAS/BV-03-C
+LL/CON/MAS/BV-04-C
+LL/CON/MAS/BV-05-C
+LL/CON/MAS/BV-07-C
+LL/CON/MAS/BV-08-C
+LL/CON/MAS/BV-09-C
+LL/CON/MAS/BV-13-C
+LL/CON/MAS/BV-20-C
+LL/CON/MAS/BV-21-C
+LL/CON/MAS/BV-23-C
+LL/CON/MAS/BV-24-C
+LL/CON/MAS/BV-25-C
+LL/CON/MAS/BV-26-C
+#LL/CON/MAS/BV-27-C #currently failing, to be investigated
+LL/CON/MAS/BV-29-C
+LL/CON/MAS/BV-30-C
+LL/CON/MAS/BV-34-C
+LL/CON/MAS/BV-35-C
+LL/CON/MAS/BV-41-C
+LL/CON/MAS/BV-43-C
+#LL/CON/MAS/BV-73-C # requires update of EDTT due to change in DLE algorithm
+#LL/CON/MAS/BV-74-C # requires update of EDTT due to change in DLE algorithm
+#LL/CON/MAS/BV-76-C # requires update of EDTT due to change in DLE algorithm
+#LL/CON/MAS/BV-77-C # requires update of EDTT due to change in DLE algorithm
+#LL/CON/SLA/BI-08-C # requires update of EDTT due to change in DLE algorithm
+LL/CON/SLA/BV-04-C
+LL/CON/SLA/BV-05-C
+LL/CON/SLA/BV-06-C
+LL/CON/SLA/BV-10-C
+LL/CON/SLA/BV-11-C
+LL/CON/SLA/BV-12-C
+LL/CON/SLA/BV-14-C
+LL/CON/SLA/BV-19-C
+LL/CON/SLA/BV-20-C
+LL/CON/SLA/BV-22-C
+LL/CON/SLA/BV-24-C
+LL/CON/SLA/BV-25-C
+LL/CON/SLA/BV-26-C
+LL/CON/SLA/BV-27-C
```

Comment:
Are these additional tests compare to being tested for legacy LLCP?

Reply:
Same tests as for legacy, but for new LLCP